lib/simp/cli/config/items/action/add_puppet_hosts_entry_action.rb

```diff
@@ -33,9 +33,9 @@ def apply
       File.readlines(@file).each do |line|
         # TODO Do we really want to remove comments?
         next if line =~ /\s*#/
-        next if line =~ /#{puppet_server}/ and @value.eql?(puppet_server)
-        next if line =~ /localdomain/
-        next if line =~ /localdomain6/
+        next if line =~ /#{puppet_server}/ && @value.eql?(puppet_server)
+        next if line =~ /^\s*127.0.0.1\s+/
```

Comment:
When we originally discussed the bug, you wanted to check for `localhost`.  Why did you decide to look for IPs instead?

Reply:
@lnemsick-simp On some reflection, `localhost` is a valid name so this is more precise (but I do need to escape my `.`